src/Domain/DonationService.php

```diff
@@ -878,7 +883,7 @@ private function shouldInviteRegistration(Donation $donation): bool
     {
         $donorId = $donation->getDonorId();
         if (!$donorId) {
-            // must be an old donation
+            // must be an already persisted donation
```

Comment:
I think old did mean old here - a donation from before we started writing to the `donorUUID` for every donation.

Reply:
Thanks, put that back how it was